daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md

```diff
@@ -0,0 +1,121 @@
+---
+type: docs
+title: "Dapr API Logs"
+linkTitle: "API Logs"
+weight: 3000
+description: "Understand how API logging works in Dapr and how to view logs"
+---
+
+API logging enables you to see the API calls from your application to the Dapr sidecar and debug issues as a result with different verbosity levels. You can also combine Dapr API logging with Dapr log events (see [configure and view Dapr Logs]({{< ref "logs-troubleshooting.md" >}}) into the output, if you want to use the logging capabilities together.
+
+## Overview
+
+API logs have different, configurable verbosity levels and is applicable for both HTTP and GRPC calls.
+The levels outlined below are the same for both system components and the Dapr sidecar process/container:
```

Comment:
Question: I presume that API logs has nothing to do with system components, such as placement and sentry services? If so then I suggest removing this sentence. Do you agree?
```suggestion
```